tests/wasmfs/wasmfs_create.c

```diff
@@ -87,6 +87,17 @@ int main() {
   printf("File contents: %s", buf3);
   assert(errno == 0);
 
+  // Try to make a file with a name that is equal to WASMFS_NAME_MAX.
+  errno = 0;
+  open("0Hh03R0h7k4lsJrgY4oVbflkHMwaqUJIKv2KJmbSwXKyY83pBaKDnq5yExUOPDA2stpqSg"
+       "DhY4UsIZsHP0ORYuxrhXwjnCMJYwkMxRc3RUjSfZU3qvPRfqwjQQCetY4rnAtJPn9D282a"
+       "bfPHi87PisnLj6dU47ZX3r4yvA7ZlsvfoZrRiopvocXOhAvVzz1QyLbIbN1cEXkk1Z5uLA"
+       "zHnD0zL276mzcC1Ir21WDLZFPJeHwOti6dpq9magNfRH8I",
```

Comment:
It would be nice if we could use a name that had some repetitive structure to make it easier for readers to determine its length. For example `000000010000000200000003...`